backend/routes/post.py

```
@@ -123,6 +124,28 @@ def get_posts(
 
     return {"posts": post_list, "count": len(post_list)}
 
+def send_post_notifications(db: Session, user: User, post: Post):
```

Ruff (0.8.2)

Missing return type annotation for public function `send_post_notifications`

Add return type annotation: `None`

(ANN201)